caluma/caluma_form/tests/test_document.py

```diff
@@ -385,6 +385,7 @@ def test_save_document(db, document, schema_executor, update):
     assert same_id == update
 
 
+@pytest.mark.parametrize("use_api", [True, False])  # noqa:C901
```

Comment:
The naming is not clear enough (also stated in an existing test). We should name it `use_graphql` or something..

Reply:
Thinking about it, we should use `use_python_api` or something. The vast majority of tests use graphql so we want to denote the exception